repositories/byod/_posts/2016-01-23-using-private-docker-images.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Using Private Docker Images
+short: Using Private Docker Images
+category: repositories
+order: 7
+---
+
+Many teams use private Docker registries to securely host images that are built from their source code. If your repositories build from private images, you can set up access to your registry so we can build environments with your images.
+
+*Note: This won’t work if you are using Docker Compose files in Runnable. Private registry support for Compose is coming soon.*
```

Comment:
Then it may be better worded as, "This does not apply if you are using Docker Compose…"